drivers/storage/libstorage/libstorage_driver_tls.go

```diff
@@ -45,49 +45,49 @@ func verifyKnownHost(
 				"actualHost":          cert.Subject.CommonName,
 				"expectedHost":        knownHost.Host,
 			}).Debug("matched tls known host information")
-			return nil
+			return true, nil
 		}
 	}
-	return errServerFingerprint
+	return false, newErrKnownHost(peerCerts)
 }
 
 func verifyKnownHostFiles(
```

Comment:
I would change those two names, `verifyKnwonHostFiles` and `verifyKnownHostsFile`, to be more differentiated or document clearly how each is different.